src/web/mjs/connectors/Yanmaga.mjs

```diff
@@ -0,0 +1,145 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Yanmaga extends Connector {
+    constructor() {
+        super();
+        super.id = 'yanmaga';
+        super.label = 'Yanmaga';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://yanmaga.jp';
+        this.apiUrl = 'https://api2-yanmaga.comici.jp';
+        this.DEFAULT_ORDER = [];
```

Comment:
May use camel case since the uppercase name seems a bit inconsistent regarding naming in this file